entrypoint.py

```diff
@@ -37,5 +37,6 @@
     owner_length = len(os.environ["GITHUB_REPOSITORY_OWNER"]) + 1
     repo_name = os.environ["GITHUB_REPOSITORY"][owner_length:]
 
+    environment_file.write(f"APP_ENV={env}\n")
```

Comment:
If I remember correctly there will still need to be some workflows that set APP_ENV... as I think a pull request has a slightly different variable it works out the env from. I'll double check, but whatever this means that would be the exception rather than having to be all over the place!